src/common/sourceMaps/turboGlobStream.ts

```diff
@@ -299,11 +301,12 @@ export class TurboGlobStream<E> {
   }
 
   private async handleFile(mtime: number, path: string, cache: CacheTree<IGlobCached<E>>) {
+    const platformPath = sep === '/' ? path : path.replace(forwardSlashRe, sep);
```

Comment:
It's the opposite of https://github.com/microsoft/vscode/blob/e06874c041d367c00da10e6034fc2db4af135ec2/src/vs/base/common/extpath.ts#L32